__tests__/app.tests.js

```diff
@@ -125,4 +125,52 @@ describe('GET /api/articles', () => {
                 expect(articles).toBeSorted('created_at', { descending: true })
             });
     });
+});
+
+describe.only('GET /api/articles/:article_id/comments', () => {
+    test('200: Should return an array of comments associated with the given article_id', () => {
+        return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then((response) => {
+                const comments = response.body.comments;
+                expect(comments).toHaveLength(11);
+                comments.forEach((comment) => {
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        article_id: 1,
+                    });
+                })
+            });
+    });
+    test('200: Should return comments sorted by created_at key order descending', () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+            .expect(200)
+            .then((response) => {
+                const comments = response.body.comments;
+                expect(comments).toHaveLength(11);
+                expect(comments).toBeSorted('created_at', { descending: true })
+            });
+    });
+    test('404: Should return an error with a message when there is no article_id or no comments associated with that article_id.', () => {
```

Comment:
If the article is found, this should be a 404 error - as this denotes that it couldn't be found. If there is an article that exists but it has no comments, this should be a 200 status code and an empty array should be returned

Reply:
In essence, these should be two different tests 

 - article id doesn't exist yet 
  - article id does exist but there are no comments